### ISSUE TITLE
Add support for aria-label in Grid

### ISSUE THREAD

**Issue by torok-peter** [vaadin/flow#3016]
Equivalent feature (part) in Fw8: https://github.com/vaadin/framework/commit/e65efdfb50bc10bedf506d475f582276f6be0e2f

May be extended to other components?

**Comment by knoobie** [comment]
Side note: Better practise would be to replace the current implement of Grid sorting. Instead of using ::after and css it would be way better to add a button element in the header cell to toggle the sorting with different captions depending on the state/sorting

@ extend to other components: 

It would be awesome if you can provide a server API to add any attribute/value to any DOM Element. So it's up to the developer or addon devs to take full advance in this fields.

**Comment by mshabarov** [comment]
Closed as it doesn't have enough context and acceptance criteria on what exactly is desired to have for Grid: aria-label for sort toggles, columns, header/footer and so on. A huge work on accessibility has been done by Vaadin Design Team and, as I know, still on-going. If someone is aware about a good use case for Grid, please report a new issue in https://github.com/vaadin/flow-components or https://github.com/vaadin/web-components.
